backends/server/package.json

```diff
@@ -5,11 +5,19 @@
   "main": "server.js",
   "scripts": {
     "test": "echo \"Error: no test specified\" && exit 1",
-    "start": "node server.js"
+    "build": "yarn tsc",
+    "start": "node server.js",
+    "dev": "yarn tsc -w & nodemon dist/index.js"
```

Comment:
Please check that lines you insert accually works ❤️
I spent 30 minutes trying figure out why the command "yarn dev" is not working...

Reply:
sorry, probable didnt save to the last changes